src/models/user.ts

```diff
@@ -373,7 +373,7 @@ export class FacilityMemberUser extends FacilityUser {
   }
 
   async getSession (params: { id: number }) {
-    const { session } = await this.action('facilitySession:show', { ...params, userId: this.id }) as SessionResponse
+    const { session } = await this.action('facilitySession:show', params) as SessionResponse
```

Comment:
Why remove the `userId` from this query?

Reply:
Because that action doesn't allow you to pass both `userId` or `id`.  You can only pass one.  Since it is using the `facilitySession` model, then we can simply pass the `id` of the session and it will return the correct session.